src/pages/AnimeKO/main.ts

```diff
@@ -0,0 +1,82 @@
+import { pageInterface } from '../pageInterface';
+
+let jsonData;
+
+export const AnimeKO: pageInterface = {
+  name: 'AnimeKO',
+  domain: 'https://animeko.co',
+  languages: ['French'],
+  type: 'anime',
+  isOverviewPage() {
+    return !!j.$('.showcase-shadow').length;
+  },
+  isSyncPage() {
+    return !!j.$('.player-wrap').length;
+  },
+  sync: {
+    getTitle(url) {
+      return j.$('.small-card h2').text().trim();
+    },
+    getIdentifier(url) {
+      return utils
+        .absoluteLink(j.$('.small-card h2 a').attr('href'), AnimeKO.domain)
+        .split('/')
+        .pop();
+    },
+    getOverviewUrl(url) {
+      return utils.absoluteLink(j.$('.small-card h2 a').attr('href'), AnimeKO.domain);
+    },
+    getEpisode(url) {
+      return Number(j.$('.player-releases :selected').text().split(' ')[1]);
+    },
+    nextEpUrl(url) {
+      return utils.absoluteLink(j.$('.btn-next').attr('href'), AnimeKO.domain);
+    },
+    getMalUrl(provider) {
+      return 'https://myanimelist.net/anime/' + jsonData.mal_id;
```

Comment:
Dangerous. An empty mal_id would result in a broken myanimelist url. Please check that not empty.